packages/salesforcedx-vscode-apex-debugger/package.nls.json

```diff
@@ -9,6 +9,6 @@
   "trace_text": "Produce diagnostic output. You can set this to true, false, or one or more selectors separated with commas. Selectors include 'all' (very detailed output; the equivalent of true), 'protocol', 'variables', 'launch', 'breakpoints' and 'streaming'.",
   "configuration_title": "Salesforce Apex Debugger Configuration",
   "connection_timeout_ms_description": "Connection timeout for Apex Debugger API requests (in milliseconds).",
-  "exception_breakpoint_command_text": "Apex Debug: Configure Exceptions",
+  "exception_breakpoint_command_text": "SFDX: Configure Apex Debug Exceptions",
```

Comment:
@hbc-bindu does this message look okay?

thank you

Reply:
Hi @cademay, looks good to me. Thanks.